### PR TITLE
Improve RNNT Loss docstrings

### PR DESCRIPTION
**Pull request by carolineechen**
contains changes from #1610 #1615, merge after

### CODE REVIEW

**Review by vincentqb — APPROVED** [review]
LGTM

File: torchaudio/prototype/rnnt_loss.py [hunk 1]
```diff
@@ -24,12 +24,13 @@ def rnnt_loss(
     dependencies.
 
     Args:
-        logits (Tensor): Tensor of dimension (batch, time, target, class) containing output from joiner
+        logits (Tensor): Tensor of dimension (batch, max seq length, max target length + 1, class)
```
Comment (carolineechen) [inline torchaudio/prototype/rnnt_loss.py:27]: TODO: also update this in line 98 for RNNTLoss
